lib/bango/errors.py

```diff
@@ -15,9 +15,23 @@ class AuthError(BangoError):
 
 class BangoFormError(BangoError):
 
-    """Something in the data we passed caused an error in the bango end."""
+    """Something in the data we passed caused an error in the Bango end."""
+
+
+class BangoImmediateError(Exception):
+
+    """
+    Something in the data we passed caused an error in Bango and rather
+    than let it get trapped somewhere, we want to raise this immediately.
+    """
 
 
 class ProxyError(Exception):
 
     """The proxy returned something we didn't like."""
+
+
+class ProcessError(Exception):
```

Comment:
Maybe ``ProcessingError`` ? Out of context it sounds weird. A docstring would be nice.